README.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1. Streaming API
 The streaming API lets you consume telemetry.
+[Consuming Idun Streaming API](https://github.com/Vasakronan/Idun-Examples/tree/master/Idun-Streaming-Api/)
```

Comment:
little confusion here: we have the same implementation example codesource with EventProcessorHost for both cersions of Streaming-api, but one marked deprecated.

Reply:
Well, before this PR is merged, that link is to the wrong thing. The current (pre-merge URL would be https://github.com/Vasakronan/Idun-Examples/tree/0.4.0-streaming-api/Idun-Streaming-Api
So it is not the same.